oktapam/client/secret_folder.go

```diff
@@ -0,0 +1,104 @@
+package client
+
+import (
```

Comment:
Not sure if we have openapi spec for secret folder apis. Can we add this to spec first and then generate sdk code ?